storagepool/storagepool.go

```diff
@@ -49,9 +55,11 @@ func storagepoolAgent(c *cli.Context) {
 		log.Fatal(err)
 	}
 
-	storagepoolAgent := NewStoragepoolAgent(healthCheckInterval, storagepoolRootDir, storagepoolUUID, healthCheckBaseDir, cattleClient)
+	storagepoolAgent := NewStoragepoolAgent(healthCheckInterval, storagepoolRootDir, storagepoolUUID, healthCheckBaseDir, healthCheckType, cattleClient)
+
+	metadataUrl := "http://rancher-metadata"
```

Comment:
Take this as an option to the program and default it to `http://rancher-metadata/07-25-2015`